pkg/stdlib/strings/fmt.go

```diff
@@ -0,0 +1,88 @@
+package strings
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/MontFerret/ferret/pkg/runtime/core"
+	"github.com/MontFerret/ferret/pkg/runtime/values"
+)
+
+// Fmt formats the template using these arguments.
+// @params template (String) - template.
+// @params args (Any Values) - template arguments.
+// @returns (String) - string formed by template using arguments.
+func Fmt(_ context.Context, args ...core.Value) (core.Value, error) {
+	err := core.ValidateArgs(args, 1, core.MaxArgs)
+	if err != nil {
+		return values.None, err
+	}
+
+	err = core.ValidateType(args[0], core.StringType)
+	if err != nil {
+		return values.None, err
+	}
+
+	formatted, err := format(args[0].String(), args[1:]...)
+	if err != nil {
+		return values.None, err
+	}
+
+	return values.NewString(formatted), nil
+}
+
+func format(template string, args ...core.Value) (string, error) {
+	rgx, err := regexp.Compile("{[0-9]*}")
+	if err != nil {
+		return "", fmt.Errorf("failed to build regexp: %v", err)
```

Comment:
Let's be consistent in terms of errors. Could you use ``https://github.com/pkg/errors`` instead of ``fmt``?